internal/acceptance/lakeview_dashboard_test.go

```diff
@@ -0,0 +1,412 @@
+		},
+		Template: `
+		resource "databricks_lakeview_dashboard" "d1" {
+			display_name			= 	"Monthly Traffic Report"
+			warehouse_id			=	"{env.TEST_DEFAULT_WAREHOUSE_ID}"
+			file_path				=	"/tmp/LakeviewDashboardTest/Dashboard.json"
+			parent_path				= 	"/Shared/provider-test"
+		}
+		`,
+		Check: resource.ComposeTestCheckFunc(
+			resource.TestCheckResourceAttr("databricks_lakeview_dashboard.d1", "display_name", "Monthly Traffic Report"),
+			func(state *terraform.State) error {
+				dashboard, ok := state.RootModule().Resources["databricks_lakeview_dashboard.d1"]
+				if !ok {
+					return fmt.Errorf("dashboard resource not found")
+				}
+				fmt.Println(dashboard.Primary.Attributes["serialized_dashboard"])
+				return nil
+			},
+			func(s *terraform.State) error {
+				dashboard, ok := s.RootModule().Resources["databricks_lakeview_dashboard.d1"]
+				if !ok {
+					return fmt.Errorf("dashboard resource not found")
+				}
+				dashboard_id = dashboard.Primary.ID
+				assert.NotEmpty(t, dashboard_id)
+				display_name = dashboard.Primary.Attributes["display_name"]
+				assert.NotEmpty(t, display_name)
+				warehouse_id = dashboard.Primary.Attributes["warehouse_id"]
+				assert.NotEmpty(t, warehouse_id)
+				etag = dashboard.Primary.Attributes["etag"]
+				assert.NotEmpty(t, etag)
+				return nil
+			},
+		),
+	}, step{
+		PreConfig: func() {
+			w, err := databricks.NewWorkspaceClient(&databricks.Config{})
+			if err != nil {
+				fmt.Println(err)
+			}
+			_, err = w.Lakeview.Update(context.Background(), dashboards.UpdateDashboardRequest{
+				DashboardId:         dashboard_id,
+				DisplayName:         display_name,
+				Etag:                etag,
+				WarehouseId:         warehouse_id,
+				SerializedDashboard: "{\"pages\":[{\"name\":\"b532570b\",\"displayName\":\"New Page Modified Remote\"}]}",
+			})
+			if err != nil {
+				fmt.Println(err)
+			}
+		},
+		Template: `
+		resource "databricks_lakeview_dashboard" "d1" {
+			display_name			= 	"Monthly Traffic Report"
+			warehouse_id			=	"{env.TEST_DEFAULT_WAREHOUSE_ID}"
+			file_path				=	"/tmp/LakeviewDashboardTest/Dashboard.json"
+			parent_path				= 	"/Shared/provider-test"
+			embed_credentials		=	false
+		}
+		`,
+		Check: resource.ComposeTestCheckFunc(
+			resource.TestCheckResourceAttr("databricks_lakeview_dashboard.d1", "display_name", "Monthly Traffic Report"),
+			func(state *terraform.State) error {
+				dashboard, ok := state.RootModule().Resources["databricks_lakeview_dashboard.d1"]
+				if !ok {
+					return fmt.Errorf("dashboard resource not found")
+				}
+				fmt.Println(dashboard.Primary.Attributes["serialized_dashboard"])
+				return nil
+			},
+		),
+	}, step{
+		Template: `
+		resource "databricks_lakeview_dashboard" "d1" {
+			display_name			= 	"Monthly Traffic Report"
+			warehouse_id			=	"{env.TEST_DEFAULT_WAREHOUSE_ID}"
+			file_path				=	"/tmp/LakeviewDashboardTest/Dashboard.json"
+			parent_path				= 	"/Shared/Teams"
+		}
```

Comment:
These templates can probably be constructed with a helper function which takes in parameters to update the corresponding fields, that will make the test look way more concise.